rust/processor/src/db/common/models/object_models/v2_objects.rs

```diff
@@ -31,6 +31,7 @@ pub struct Object {
     pub guid_creation_num: BigDecimal,
     pub allow_ungated_transfer: bool,
     pub is_deleted: bool,
+    pub untransferrable: bool,
```

Comment:
nit: typo -> untransferable

Reply:
actually neither is a word haha. i prefer 2 r's but idk 🤷